hkm/templates/hkm/base.html

```diff
@@ -65,11 +65,13 @@
   <!-- iOS web app: Specifies a splash image to use when the page is loading. -->
   <link rel="apple-touch-startup-image" href="ios-splash.png">
 
-  <!-- iOS web app: icons. -->
+  <!-- iOS web app: icons. These are missing, just comment them out for now to stop errors appearing -->
+  <!--
   <link rel="apple-touch-icon-precomposed" href="ios-app-icon-57.png">
   <link rel="apple-touch-icon-precomposed" sizes="72x72" href="ios-app-icon-72.png">
   <link rel="apple-touch-icon-precomposed" sizes="114x114" href="ios-app-icon-114.png">
   <link rel="apple-touch-icon-precomposed" sizes="144x144" href="ios-app-icon-144.png">
+  -->
```

Comment:
We could probably just remove these lines altogether. If we do need those at some point, we can probably add them when we know which icons we should show.

Reply:
Removed.